doc/en/logging.rst

```diff
@@ -224,6 +224,8 @@ made in ``3.4`` after community feedback:
 
 * Log levels are no longer changed unless explicitly requested by the :confval:`log_level` configuration
   or ``--log-level`` command-line options. This allows users to configure logger objects themselves.
+  When this parameter is used and user requests log level of smaller numeric value than the log level of the root
+  logger then the root logger will be adjusted to have level equal to the requested logging level.
```

Comment:
In case this doesn't make sense please suggest rephrase!

Reply:
I think we don't need to document it in logging.rst. I think having a reasonably descriptive msg in the changelog is enough.

Reply:
Removed.